drivers/eeprom/spi_xx25xx.c

```diff
@@ -392,6 +393,7 @@ static void ee25xx_waitwritecomplete(struct ee25xx_dev_s *priv)
       /* Deselect the FLASH */
 
       SPI_SELECT(priv->spi, SPIDEV_EEPROM(0), false);
+      ee25xx_unlock(priv->spi);
```

Comment:
Maybe can additionally change the function prototype to take `spi` as a parameter and not `priv` since nothing except `priv->spi` is used in the function body.